python/ray/tune/examples/sigopt_example.py

```diff
@@ -0,0 +1,70 @@
+"""This test checks that SigOpt is functional.
+
+It also checks that it is usable with a separate scheduler.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+from ray.tune import run_experiments, register_trainable
+from ray.tune.schedulers import AsyncHyperBandScheduler
+from ray.tune.suggest import SigOptSearch
+
+
+def easy_objective(config, reporter):
+    import time
+    time.sleep(0.2)
+    for i in range(config["iterations"]):
+        reporter(
+            timesteps_total=i,
+            neg_mean_loss=-(config["height"] - 14)**2 +
+            abs(config["width"] - 3))
+        time.sleep(0.02)
+
+
+if __name__ == '__main__':
+    import argparse
+
```

Comment:
in this example, can you also add a check for the key? (like an assertion)

Reply:
should i make a check for the key in sigopt.py too?